rules/windows/c2_installutil_beacon.toml

```
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies InstallUtil.exe making outbound network connections. This may indicate adversarial activity as InstallUtil is
+often leveraged by adversaries to execute code and evade detection.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "InstallUtil Making Network Connections"
```

```suggestion
name = "InstallUtil Network Connection Sequence"
```

---

I think it's good to denote that this rule doesn't just find a network connection, but that it has a process _and_ a network connection.

I'm not sold on this name yet, mostly because it's a bunch of nouns together, and it's not obvious from the name what the intent is. Is it a "InstallUtil Network Connection" Sequence? or a "InstallUtil Network" "Connection Sequence"?

This is pretty literal about what the rule detects. I'm not sure if that's goo dor bad
```suggestion
name = "InstallUtil Process with Network Connections"
```

---

```suggestion
name = "InstallUtil Process Making Network Connections"
```